File: src/com/Torvald/ColourUtil/LimitedColours.java
package com.Torvald.ColourUtil;

import org.newdawn.slick.Color;

/**
 * Created by minjaesong on 16-02-11.
 */
public interface LimitedColours {



    Color createSlickColor(int raw);
    Color createSlickColor(int r, int g, int b);

    void create(int raw);
    void create(int r, int g, int b);

}

File: src/com/Torvald/ColourUtil/LimitedColours.kt
package com.torvald.colourutil

import org.newdawn.slick.Color

/**
 * Created by minjaesong on 16-02-11.
 */
interface LimitedColours {

    fun createSlickColor(raw: Int): Color
    fun createSlickColor(r: Int, g: Int, b: Int): Color

    fun create(raw: Int)
    fun create(r: Int, g: Int, b: Int)

    fun toSlickColour(): Color
}
